test/e2e/azure_lb.go

```diff
@@ -122,10 +123,10 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 
 	// TODO: fix and enable this. Internal LBs + IPv6 is currently in preview.
 	// https://docs.microsoft.com/en-us/azure/virtual-network/ipv6-dual-stack-standard-internal-load-balancer-powershell
-	if !input.IPv6 {
+	if !isIPv6 {
```

Comment:
I think we should be able to remove this check as IPv6 ILB is supported: https://github.com/kubernetes-sigs/cloud-provider-azure/pull/703. I can remove this check in a follow-up.